sdk/tables/data-tables/src/generated/operations/service.ts

```diff
@@ -20,9 +20,8 @@ import {
   ServiceGetStatisticsResponse
 } from "../models";
 
-/**
- * Class representing a Service.
- */
+/** Class representing a Service. */
+/** @hidden */
```

Comment:
This is manually added for this PR. I will do the Generator Code change soon.

Reply:
Can you please add a comment that this is manually added and link the issue tracking the generator changes? Unless you plan to merge this after the generator changes are merged

Reply:
Here is the issue - https://github.com/Azure/autorest.typescript/issues/849. Will send in the fix today.